native/src/main/java/io/ballerina/stdlib/ftp/server/FtpListener.java

```diff
@@ -77,23 +80,45 @@ public boolean onMessage(RemoteFileSystemBaseMessage remoteFileSystemBaseMessage
             RemoteFileSystemEvent event = (RemoteFileSystemEvent) remoteFileSystemBaseMessage;
             Map<String, Object> watchEventParamValues = processWatchEventParamValues(event);
             if (runtime != null) {
+                Callback callback = new Callback() {
+                    @Override
+                    public void notifySuccess(Object o) {
+                    }
+                    @Override
+                    public void notifyFailure(BError error) {
+                        log.error("Error while invoking FTP onMessage method.");
+                    }
```

Comment:
Don't we need to print the stracktrace if there is any error ?

Reply:
This was how it was implemented previously also. Do we need to change that?
WDYT @Bhashinee ?

Reply:
Let's print the stack trace as @ayeshLK mentioned. 
Without logging let's do
`error.printStackTrace();`

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-ftp/pull/580/commits/4d55d496c675cc8e6e781ccde3dd63d0dfbae198